index.js

```diff
@@ -1,25 +1,49 @@
+/* eslint-disable no-var, prefer-template, comma-dangle */
+
 'use strict';
 
 module.exports = function buildAirbnbPreset(context, options) {
+  // This is similar to how `env` works in Babel:
+  // https://babeljs.io/docs/usage/babelrc/#env-option
+  // We are not using `env` because it’s ignored in versions > babel-core@6.10.4:
+  // https://github.com/babel/babel/issues/4539
+  // https://github.com/facebookincubator/create-react-app/issues/720
+  // It’s also nice that we can enforce `NODE_ENV` being specified.
+  var env = process.env.BABEL_ENV || process.env.NODE_ENV || 'development';
+
+  var allPlugins = [
+    [require('babel-plugin-transform-es2015-block-scoping'), { throwIfClosureRequired: true }],
```

Comment:
What i mean is, it's ok to add this as an option (like "modules" [here](https://github.com/airbnb/babel-preset-airbnb/pull/9/files#diff-168726dbe96b3ce427e7fedce31bb0bcR27)) but it can't be on by default or else it'd be a breaking change.